english/wake-up/wake-up-again/wake-up-again.md

```diff
@@ -10,4 +10,4 @@ figured you’d be able to go back to sleep. But not back to the nightmare
 
 you just had, obviously.
 
-As your eyes droop and you started drifting back into the other world
```

Comment:
This modification is fine. If you break up the long line, I'll be happy to merge this.